src/components/analytics/hooks/useAnalyticsReporter.ts

```diff
@@ -71,6 +72,10 @@ export function useAnalyticsReporter() {
     // custom dimension 2 - walletname
     googleAnalytics.setDimension(Dimensions.walletName, account ? walletName : 'Not connected')
 
+    // Set userId and also dimensions because ReactGA.set might not be working
+    googleAnalytics.setDimension(Dimensions.userAddress, account)
+    ReactGA.set({ userId: account })
```

Comment:
Aren't you sure it works?
How can you test it?

Reply:
That seems to be code before the [second commit](https://github.com/cowprotocol/cowswap/pull/2450/commits/36f05c6f7df80e7e041b776a2793f84b0fa0d95f), now it looks like this 
![Screenshot 2023-05-15 at 10 11 51](https://github.com/cowprotocol/cowswap/assets/34926005/05b64daf-0d49-4872-b741-59960b2a4c2f)

I've tested it in GA dashboard but if its not passed with quotes we get the first item like here
![Screenshot 2023-05-15 at 10 14 43](https://github.com/cowprotocol/cowswap/assets/34926005/af4a7f26-27e6-4767-915b-0119e7a68771)

Reply:
In that case (if it doesn't work), we might not use `ReactGA.set` at all, no?